static/app/views/organizationGroupDetails/header.tsx

```diff
@@ -295,177 +308,178 @@ class GroupHeader extends Component<Props, State> {
           to={`${baseUrl}tags/${location.search}`}
           isActive={() => currentTab === Tab.TAGS}
           disabled={disabledTabs.includes(Tab.TAGS)}
-          onClick={() => this.tabClickAnalyticsEvent(Tab.TAGS)}
+          onClick={() => tabClickAnalyticsEvent(Tab.TAGS)}
         >
           {t('Tags')}
         </ListLink>
         <ListLink
           to={eventRouteToObject}
           isActive={() => currentTab === Tab.EVENTS}
           disabled={disabledTabs.includes(Tab.EVENTS)}
-          onClick={() => this.tabClickAnalyticsEvent(Tab.EVENTS)}
+          onClick={() => tabClickAnalyticsEvent(Tab.EVENTS)}
         >
           {t('Events')}
         </ListLink>
       </Fragment>
     );
+  }, [
+    baseUrl,
+    currentTab,
+    disabledTabs,
+    group.numComments,
+    location.query,
+    location.search,
+    tabClickAnalyticsEvent,
+  ]);
+
+  const membersList = useMemnberList({group, organization});
+  const {userCount} = group;
+
+  let className = 'group-detail';
+
+  if (group.hasSeen) {
+    className += ' hasSeen';
```

Comment:
I know this is just copy/pasted from what it was before but ick, more class names from less files?

Reply:
Yeah I gotta look closer